### ISSUE TITLE
Transition over to focusing on XeTeX

### ISSUE THREAD

**Issue by benyakirten** [benyakirten/latex-parser-ts#13]
XeTeX allows modern font systems, which will be absolutely necessary to make progress.

**Comment by RedWilly** [comment]
is this still an active project

if so do you have plans to published an npm package

**Comment by benyakirten** [comment]
No, this is no longer actively worked on, nor is it complete. I've re-evaluated my needs, and I will be rewriting it in Rust when another project gets to the stage that I need it. However, in the meantime, you're free to take any code that's useful to you. If you'd like, I can try to tell you what my plans or sources are.
